app/webpacker/controllers/mailing-list-popup_controller.js

```diff
@@ -9,6 +9,7 @@ export default class extends Controller {
   topScrollMinimumDistance = 700;
 
   connect() {
+    console.log("HERE")
```

Comment:
Probably we don't need this one?

Reply:
Oh whoops, left that in from a previous PR I was debugging! Nice spot, I'll remove it now